src/app/map.service.ts

```diff
@@ -0,0 +1,91 @@
+import { Injectable } from '@angular/core';
```

Comment:
split this service to 2 files: one for http related function and one for map related functions